modules/tchap-translations/tchap_translations.json

```
@@ -663,12 +663,12 @@
         "fr": "La connexion à Tchap n'est pas possible pour le moment (err:03). Voir l'état du service: https://status.tchap.numerique.gouv.fr"
     },
     "threads|unable_to_decrypt": {
-        "en": "Waiting for this message. This could take a while.",
-        "fr": "En attente de ce message. Ceci pourrait prendre un moment."
+        "en": "Waiting for this message to be decrypted...",
```

On peut même faire plus court :
```suggestion
        "en": "Decrypting...",
```